admin/class-create-block-theme-admin.php

```diff
@@ -729,9 +720,6 @@ function make_media_blocks_local ( $nested_blocks ) {
 				case 'core/media-text':
 					$block = $this->make_mediatext_block_local( $block );
 					break;
-				case 'core/template-part':
-					$block = $this->add_theme_attr_to_template_part_block( $block );
-					break;
```

Comment:
A theme attribute is still attached to template parts.
```
<!-- wp:template-part {"slug":"header","tagName":"header","theme":"themes/blockbase2"} /-->
```

Reply:
What's the problem with that? The template parts are loaded fine, I don't know why they shouldn't have the theme key.